src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,240 @@
+
+In the file, find the values for the following keys:
+
+- `client_id`
+- `client_secrets`
+- `technical_account_email`
+- `technical_account_id`
+- `ims_org_id`
+
+Create a GitHub secret for each of these items.
+
+You also need to specify the `id` for the organization, project, and workspace in the following section of GitHub workflow. These values are also available in the file downloaded from the Developer Console:
+
+```yml
+- name: Select org
+  run: aio console:org:select 123456
+- name: Select project
+  run: aio console:project:select 1234567890
+- name: Select workspace
+  run: aio console:workspace:select 12345678901
+```
+
+Repeat this process in your local environment, by adding these values as [environment variables](./developer-tools.md#environment-variables).
+
+This CI/CD process corresponds to a single workspace. If you want to create another workflow, for example, to differentiate between `stage` and `prod`, you need to:
+
+-  Duplicate the workflow you created
+-  Replace the workspace `id`
+
+## 4. Create a GitHub action
+
+The GitHub action described in this section updates the specified mesh when a PR is merged to the `main` branch of your repository. This step uses the variables and secrets you configured in the previous section.
+
+This workflow relies on two existing Adobe I/O GitHub Actions published on the GitHub Marketplace:
+
+-  [`adobe/aio-cli-setup-action`](https://github.com/adobe/aio-cli-setup-action) installs and configures the [CLI](https://github.com/adobe/aio-cli) on the GitHub infrastructure. For API Mesh, this action handles authentication.
+
+- [`adobe/aio-apps-action`](https://github.com/adobe/aio-apps-action) centralizes support for a GitHub workflow to leverage application-specific commands, such as deploying via `aio api-mesh:update`.
+
+In the `.github/workflows` folder, create a `.yml` file with the following contents:
+
+```yml
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  # push:
+  #   branches: [ "main" ]
+  # pull_request:
+  #   branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+   deploy:
+    name: Deploy to Prod
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 1
+      matrix:
+        node-version: ['20']
+        os: [ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install
```

Comment:
@jhadobe I realized that we do not need an npm install. You can remove this step.